src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,45 @@
-export const Tabs = () => {};
+import { Dispatch, SetStateAction } from 'react';
+
+type Props = {
+  tabs: Tabs[],
+  onTabSelected: Dispatch<SetStateAction<string>>,
+  selectedTabId: string,
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  return (
+    <>
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map((tab) => (
+            <li
+              key={tab.id}
+              className={selectedTab.id === tab.id ? 'is-active' : ''}
```

Comment:
I suggest using the following resource next time so that you avoid applying ternary and empty string
https://www.npmjs.com/package/classnames